app/enterprise/1.3-x/brain-immunity/troubleshooting.md

```diff
@@ -51,16 +55,16 @@ When the Collector App cannot communicate with Kong, the **kong_status** object
 ### Collector App is not able to communicate with Kong
 When the **/status** endpoint is not returning a **kong_status** object, there are two possible reasons. First, it could be the Kong Admin environment variable for the Collector App is not set properly. The environment variables are:
```

Comment:
```suggestion
When the `/status` endpoint is not returning a `kong_status` object, there are two possible reasons. First, it could be the Kong Admin environment variable for the Collector App is not set properly. The environment variables are:
```